### ISSUE TITLE
Turn on asset compilation in Production

### ISSUE THREAD

**Issue by varunrau** [calblueprint/worldreader#210]
Reasons for turning it off for now:

1. It is much easier to debug our client-side code when it has been compiled through Uglifier.
2. Although we are pushing to a production server, our app is not truly in production as we have not released it to Worldreader's partners yet.
3. Since the app is still small, we are not getting hit by any performance problems yet. (I don't forsee this being an issue anytime soon either.)

In short, I think the developer benefits far outweigh the user costs. However, once we have reached a reasonable level of stability, we should turn it back on.
